modules/nested-accordion/assets/scss/frontend.scss

```diff
@@ -0,0 +1,33 @@
+.elementor-widget-n-accordion {
+	--n-accordion-title-font-size: 15px;
+	--n-accordion-title-padding: 10px 15px;
+	--n-accordion-border-width: 1px;
+	--n-accordion-border-color: #{$gray-light};
+	--n-accordion-border-style: solid;
+
+	details {
+
+		summary {
```

Comment:
@davseve Should we not rather use class names here? What if users add `<summary>` and `<detail>` elements into the content area?

Reply:
Great suggestion

Reply:
Done